mkdocs.yml

```diff
@@ -218,17 +218,18 @@ nav:
       - data/ampli/middleware.md
       - data/ampli/migration.md
     - SDKs:
-      - TypeScript (Beta): data/sdks/typescript-browser.md
+      - TypeScript: data/sdks/typescript-browser.md
```

Comment:
This combination of Languages with Platforms is confusing to me. What if we only showed Platforms at the top-level and then put the language selection under that.

```
- Browser: data/sdks/typescript-browser.md
- Browser using Ampli: data/sdks/browser-ampli.md
- Browser (Legacy): data/sdks/javascript.md
- Browser using Ampli (Legacy): data/sdks/browser-ampli-legacy.md
- Node.js using Ampli: data/sdks/nodejs-ampli.md
- Node.js: data/sdks/node.md
- Android (Beta): data/sdks/android-kotlin.md
- Android: data/sdks/android.md
- Android using Ampli: data/sdks/android-ampli.md
```

Reply:
cc @yuhao900914 @caseyamp